certora/helpers/MockDllFifo.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../munged-fifo/DoubleLinkedList.sol";
+
+contract MockDllFifo {
+    using DoubleLinkedList for DoubleLinkedList.List;
+
+    // VERIFICATION INTERFACE
+
+    DoubleLinkedList.List public dll;
+
+    uint256 public maxIterations;
+
+    uint256 internal dummy_state_variable;
+
+    function dummy_state_modifying_function() public {
+        // to fix a CVL error when only one function is accessible
```

Comment:
is it still needed?

Reply:
Nice catch, it's not needed anymore -> removed in ac6cfd2